internal/locate/region_request.go

```diff
@@ -35,9 +35,9 @@
 package locate
 
 import (
-	"bytes"
 	"context"
 	"fmt"
+	"maps"
```

Comment:
> package maps is not in GOROOT (/opt/hostedtoolcache/go/1.18.10/x64/src/maps)

Looks like this package doesn't exist in go 1.18

Reply:
Nice catch, done.